features/unattached_status.feature

```diff
@@ -32,16 +32,18 @@ Feature: Unattached status
     @uses.config.machine_type.lxd.container
     Scenario Outline: Unattached status in a ubuntu machine
         Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I run `sed -i 's/contracts.can/contracts.staging.can/' /etc/ubuntu-advantage/uaclient.conf` with sudo
         When I run `ua status` as non-root
         Then stdout matches regexp:
             """
             SERVICE       AVAILABLE  DESCRIPTION
             cc-eal        <cc-eal>    +Common Criteria EAL2 Provisioning Packages
-            cis           <cis>       +Center for Internet Security Audit Tools
-            esm-infra     <infra>     +UA Infra: Extended Security Maintenance \(ESM\)
+            ?<cis>( +<cis-available> +Security compliance and audit tools)?
```

Comment:
Oooh sneaky TIL about full line optional regexes in behave. Nice one

Reply:
This is work from the regex magician which helped me last time I was stuck, and then he wrote this. His name is @orndorffgrant I guess not sure if you heard about him (: